x-pack/solutions/observability/test/api_integration_deployment_agnostic/apis/slo/update_slo.ts

```diff
@@ -90,5 +90,22 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
       await transformHelper.assertExist(getSLOTransformId(sloId, 2));
       await transformHelper.assertExist(getSLOSummaryTransformId(sloId, 2));
     });
+
+    it('updates dashboard artifacts and returns them', async () => {
```

Comment:
🍰 nit: can you create two sub describe (one for with revision bump and one without). This test belongs in the without revision bump. I want to make sure we document (by test) the behaviour of updating a dashboard artifacts: 
- only save the definition with the dashboard
- no revision bump (maybe assert for this)?

Reply:
Is this what you had in mind?
https://github.com/elastic/kibana/pull/232583/commits/d757b3c416ce663b582100831781e675828d268a